### PR TITLE
fix several typos under scheduler framework

### PR DESCRIPTION
**Pull request by dengyufeng2206**
Signed-off-by: dengyufeng2206 <deng.yufeng@zte.com.cn>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Word modification
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by sanposhiho — COMMENTED** [review]
/lgtm

**Comment by kerthcet** [comment]
@dengyufeng2206 You need to rebase the PR first.
/retitle fix several typos under scheduler framework

**Comment by dengyufeng2206** [comment]
> @dengyufeng2206 You need to rebase the PR first. /retitle fix several typos under scheduler framework

OK，thank you！

File: test/integration/framework/controlplane_utils.go [hunk 1]
```diff
@@ -96,4 +96,4 @@ func SharedEtcd() *storagebackend.Config {
 	cfg := storagebackend.NewDefaultConfig(path.Join(uuid.New().String(), "registry"), nil)
 	cfg.Transport.ServerList = []string{GetEtcdURL()}
 	return cfg
-}
+}
```
Comment (kerthcet) [inline test/integration/framework/controlplane_utils.go:99]: revert this plz.

Comment (dengyufeng2206) [reply test/integration/framework/controlplane_utils.go]: ok, thank you

**Comment by kerthcet** [comment]
Hi @dengyufeng2206 please squash the commits. Thanks.

**Comment by dengyufeng2206** [comment]
> Hi @dengyufeng2206 please squash the commits. Thanks.

ok，I have deleted the local branch, I'll resubmit later

**Comment by dengyufeng2206** [comment]
> Hi @dengyufeng2206 please squash the commits. Thanks.

new commit，https://github.com/kubernetes/kubernetes/pull/112028，thank you！

**Pull request closed without merging**